RxGesture.podspec

```diff
@@ -29,8 +29,8 @@ Pod::Spec.new do |s|
   s.ios.source_files      = 'Pod/Classes/iOS/*.swift'
   s.osx.source_files      = 'Pod/Classes/OSX/*.swift'
   
-  s.dependency 'RxSwift', '~> 3.6'
-  s.dependency 'RxCocoa', '~> 3.6'
+  s.dependency 'RxSwift', '~> 4.0.0-alpha.0'
+  s.dependency 'RxCocoa', '~> 4.0.0-alpha.0'
```

Comment:
I'm not a pod expert but wouldn't it work with `'RxSwift', '~> 4.0'`? I'm not sure it includes alpha/beta releases.

Reply:
CocoaPods doesn't resolve "4.0.0-beta.0" as satisfying "~> 4.0" so I think "~> 4.0.0-beta.0" is the best we can do for now.